prompt2model/dataset_retriever/column_selection_prompt.py

```diff
@@ -0,0 +1,95 @@
+        This dataset has the following columns: [sentences, questions, answers, options]""",  # noqa: E501
+        """{
+        "input": ["context", "question"],
+        "output": ["answer"],
+        "irrelevant": [],
+        "ambiguous": ["options"]
+        }""",
+    ),
+    (
+        """Your job is to be able to translate between languages. You will use the Opus100 dataset from HuggingFace. A sample data instance from this is as follows:  {"translation":{ "ca": "El department de bombers té el seu propi equip d'investigació.", "en": "Well, the fire department has its own investigative unit." }}. This dataset has the following columns: [translation]. """,  # noqa: E501
+        """{
+        "input": [],
+        "output": [],
+        "irrelevant": []
+        "ambiguous": ["translation"]
+        }""",
+    ),
+]
+
+
+def truncate_row(example_row: dict) -> str:
+    """Add truncation if the example row is too big."""
+    truncated_row = {}
+    for key in example_row.keys():
+        truncated_row[key] = json.dumps(example_row[key])[:50]
+    return json.dumps(truncated_row)
+
+
+def build_input(
+    instruction: str, dataset_name: str, dataset_columns: str, sample_row: dict
+):
+    """Template function to build input based on arguments."""
+    input_prompt_template = f"""You are tasked with the following process. {instruction} For this task, you will use the {dataset_name} dataset from HuggingFace.  The following is a sample data instance from this is as follows. {truncate_row(sample_row)}.
```

Comment:
I suggested a typo fix. Also, can we make `INPUT_PROMPT_TEMPLATE` a module-level variable like `METAPROMPT_BASE`? That will make our handling of these variables more consistent and make it more clear that it is something configurable.
```suggestion
    input_prompt_template = f"""You are tasked with the following process. {instruction} For this task, you will use the {dataset_name} dataset from HuggingFace.  A sample data instance from this is as follows. {truncate_row(sample_row)}.
```